pipeline-adapters/mlrun-pipelines-kfp-v1-8/src/mlrun_pipelines/mixins.py

```diff
@@ -94,3 +95,15 @@ def resolve_project_from_workflow_manifest(self, workflow_manifest):
                     raise NotImplementedError(f"Unknown action: {action}")
 
         return mlrun.mlconf.default_project
+
+    @staticmethod
+    def resolve_error_from_pipeline(pipeline):
+        if pipeline.run.status in ["Error", "Failed"]:
```

Comment:
```suggestion
        if pipeline.run.status in ["Error", "Failed"]:
```

use mlrun_pipelines.comon.models.RunStatuses